src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -14277,14 +14277,15 @@ def test_aks_approuting_update(self, resource_group, resource_group_location):
 
         # create cluster with app routing enabled
         create_cmd = (
-            "aks create --resource-group={resource_group} --name={aks_name} --location={location} --kubernetes-version 1.30 "
+            "aks create --resource-group={resource_group} --name={aks_name} --location={location} --kubernetes-version 1.30.0 "
```

Comment:
It's risky to hardcode a version number. Can you use the get-versions API to filter out the version you want?

Reply:
done!